src/endpoints/EndpointBuilder.js

```diff
@@ -0,0 +1,47 @@
+import Endpoint from './Endpoint'
+
+class EndpointBuilder {
+  _name
+  _url
+  _mapper
+  _stateMapper
+  _refetchLogic
+
+  constructor (name) {
+    this._name = name
+    this._stateMapper = (state) => ({
+      language: state.router.params.language,
+      location: state.router.params.location
+    })
+
+    this._refetchLogic = (options, nextOptions) => options.language !== nextOptions.language
```

Comment:
I don't like that the stateMapper has a default value.
Although I'd like the default refetchLogic to do just a deepComparison between options and nextOptions 🤔 Is that even possible? :D

Reply:
Hm maybe that's now possible, right. Previsouly options contained stuff like dispatch or t from the components properties.

Reply:
No, it didn't. There were already the options in a separate property of the fetcher.